deploy/rinkeby/007_dai_handler.ts

```diff
@@ -0,0 +1,83 @@
+import { hardhatArguments } from "hardhat";
+import { deployments } from "hardhat";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const DAIVaultHandler = async (hre: HardhatRuntimeEnvironment) => {
+	let initial_run = process.env.INITIAL_RUN == "true" ? true : false;
+	if (hardhatArguments.network === "rinkeby" && initial_run) {
+		const { log } = deployments;
+		const namedAccounts = await hre.getNamedAccounts();
+		const deployer = namedAccounts.deployer;
+		const ethers = hre.ethers;
+		const [owner] = await ethers.getSigners();
+
+		let handlerContract;
+		let orchestrator = await deployments.get("Orchestrator");
+		let ctx = await deployments.get("Ctx");
+		try {
+			handlerContract = await deployments.get("DAIVaultHandler");
+		} catch (error) {
+			try {
+				let tcap = await deployments.get("TCAP");
+
+				let DAIContract = await deployments.get("DAI");
+
+				let divisor = process.env.DIVISOR as string;
+				let ratio = process.env.RATIO as string;
```

Comment:
Just want to make a note that we need to be careful while picking these values from `.env` as these values with change for hard mode DAI.

Reply:
Or this deployment intended for DAI vaults with VR = 200%